base/field/FairRKPropagator.cxx

```diff
@@ -180,55 +391,55 @@ Double_t FairRKPropagator::OneStepRungeKutta(Double_t charge, Double_t step,
   /// *                *
   /// ******************************************************************
 
-  Double_t h2, h4, f[4];
-  Double_t xyzt[3], a=0, b=0, c=0, ph,ph2;
-  Double_t secxs[4],secys[4],seczs[4]; //hxp[3];
-  Double_t /*g1 , g2, g3, g4, g5, g6,*/ ang2, dxt, dyt, dzt;
-  Double_t est, at, bt, ct, cba;
-//  Double_t /*f1, f2, f3, f4, rho, tet, hnorm, hp, rho1, sint, cost*/;
-
-  Double_t x=0;
-  Double_t y=0;
-  Double_t z=0;
-
-  Double_t track_length = 0.;
-
-  Double_t xt;
-  Double_t yt;
-  Double_t zt;
-
-  Double_t maxit = 1992;
-  Double_t maxcut = 11;
-
-  const Double_t hmin   = 1e-4; // !!! MT ADD,  should be member
-  const Double_t kdlt   = 1e-6; // !!! MT CHANGE from 1e-4, should be member
-  const Double_t kdlt32 = kdlt/32.;
-  const Double_t kthird = 1./3.;
-  const Double_t khalf  = 0.5;
-  const Double_t kec    = 2.99792458e-4;
-  const Double_t kpisqua = 9.86960440109;
-  /*  const Int_t kix  = 0;
-    const Int_t kiy  = 1;
-    const Int_t kiz  = 2;
-    const Int_t kipx = 3;
-    const Int_t kipy = 4;
-    const Int_t kipz = 5;
+  double h2, h4, f[4];
+  double xyzt[3], a=0, b=0, c=0, ph,ph2;
+  double secxs[4],secys[4],seczs[4]; //hxp[3];
+  double /*g1 , g2, g3, g4, g5, g6,*/ ang2, dxt, dyt, dzt;
+  double est, at, bt, ct, cba;
+//  double /*f1, f2, f3, f4, rho, tet, hnorm, hp, rho1, sint, cost*/;
+
+  double x=0;
+  double y=0;
+  double z=0;
+
+  double track_length = 0.;
+
+  double xt;
+  double yt;
+  double zt;
+
+  double maxit = 1992;
+  double maxcut = 11;
+
+  const double hmin   = 1e-4; // !!! MT ADD,  should be member
+  const double kdlt   = 1e-6; // !!! MT CHANGE from 1e-4, should be member
+  const double kdlt32 = kdlt/32.;
+  const double kthird = 1./3.;
+  const double khalf  = 0.5;
+  const double kec    = 2.99792458e-4;
+  const double kpisqua = 9.86960440109;
+  /*  const int kix  = 0;
+    const int kiy  = 1;
+    const int kiz  = 2;
+    const int kipx = 3;
+    const int kipy = 4;
+    const int kipz = 5;
   */
   // *.
   // *.    ------------------------------------------------------------------
   // *.
   // *             this constant is for units cm,gev/c and kgauss
   // *
-  Int_t iter = 0;
-  Int_t ncut = 0;
-  for(Int_t j = 0; j < 7; j++) {
+  int iter = 0;
+  int ncut = 0;
+  for(int j = 0; j < 7; j++) {
     vout[j] = vect[j];
   }
 
-  Double_t pinv   = kec * charge / vect[6];
-  Double_t tl = 0.;
-  Double_t h = step;
-  Double_t rest;
+  double pinv   = kec * charge / vect[6];
+  double tl = 0.;
+  double h = step;
+  double rest;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'rest' can be reduced.](https://app.codacy.com/app/FairRootGroup/FairRoot/pullRequest?prid=4139720)